data/api/server-server/invites-v1.yaml

```diff
@@ -11,157 +11,186 @@
-                    }
+                "$ref": "examples/minimal_pdu.json",
+                "type": "m.room.member",
+                "state_key": "@joe:elsewhere.com",
+                "origin": "example.org",
+                "origin_server_ts": 1549041175876,
+                "sender": "@someone:example.org",
+                "content": {
+                    "membership": "invite"
+                },
+                "signatures": {
+                  "example.com": {
+                    "ed25519:key_version": "SomeSignatureHere"
                   },
-                  "content": {
-                      "membership": "invite"
-                  },
-                  "signatures": {
-                    "example.com": {
-                      "ed25519:key_version": "SomeSignatureHere"
-                    },
-                    "elsewhere.com": {
-                      "ed25519:k3y_versi0n": "SomeOtherSignatureHere"
-                    }
-                  }
                 }
               }
-            ]
-        403:
-          description: |-
-            The invite is not allowed. This could be for a number of reasons, including:
+        required: true
+      responses:
+        "200":
+          description: >-
+            The event with the invited server's signature added. All other
+            fields of the events
+
+            should remain untouched. Note that events have a different format depending on the
+
+            room version - check the [room version specification](/rooms) for precise event formats.
+          content:
+            application/json:
+              schema:
+                type: array
+                minItems: 2
+                maxItems: 2
+                items:
+                  - type: integer
+                    description: The value `200`.
+                    examples:
+                      - 200
+                  - type: object
+                    description: An object containing the signed invite event.
+                    title: Event Container
+                    properties:
+                      event:
+                        $ref: definitions/invite_event.yaml
+                    required:
+                      - event
+              examples:
+                response:
+                  value:
+                    [
+                      200,
+                      {
+                        "event":  {
+                          "$ref": "examples/minimal_pdu.json",
+                          "type": "m.room.member",
+                          "state_key": "@someone:example.org",
+                          "origin": "example.org",
+                          "origin_server_ts": 1549041175876,
+                          "sender": "@someone:example.org",
+                          "unsigned": {
+                            "invite_room_state": {
+                              "$ref": "../../event-schemas/examples/invite_room_state.json"
```

Comment:
The validator complains about not being able to load this reference, although it is able to resolve the exact same reference in the `requestBody`, so there must be something wrong with the syntax.

Reply:
hrm, odd. Let's park this for now and come back to it.

Reply:
The reason why it doesn't work might be because `$ref` are not supposed to be supported in examples, however that doesn't explain why it doesn't complain about other references.